src/Paysera/WalletApi/Mapper.php

```diff
@@ -1435,6 +1438,9 @@ public function decodeStatementParty($data)
         if (!empty($data['display_name']) && empty($data['name'])) {
             $this->setProperty($party, 'name', $data['display_name']);
         }
+        if (!isset($data['ibans'])) {
+            $this->setProperty($party, 'ibans', $data['ibans']);
```

Comment:
There can be cases, when there is no iban, is it ok?

Also question, how can it be, that in case it's Statement Party, then it has just one account_number, but multiple ibans?

Reply:
Well one account_number but it might resolve to multiple ibans. Iban and some aliases. It was questioned in another ticket to return all ibans along with aliases. So let's leave it as it is